README.md

```diff
@@ -42,15 +43,32 @@ $ make && sudo make install
 
 You can also use `make binaries-all` to prepare `buildkitd.containerd_only` and `buildkitd.oci_only`.
 
-`examples/buildkit*` directory contains scripts that define how to build different configurations of BuildKit and its dependencies using the `client` package. Running one of these script generates a protobuf definition of a build graph. Note that the script itself does not execute any steps of the build.
+#### Starting the buildkitd daemon:
+
+```
+buildkitd --debug --root /var/lib/buildkit
+```
+
+The buildkitd daemon suppports two worker backends: OCI (runc) and containerd.
+
+By default, the OCI (runc) worker is used.
+You can set `--oci-worker=false --containerd-worker=true` to use the containerd worker.
+
+We are open to adding more backends.
+
+#### Exploring LLB
+
```

Comment:
Can we have this image here, so that people can visually understand the concept of the graph?

https://image.slidesharecdn.com/buildkitossna-170920080230/95/state-of-builder-and-buildkit-by-tonis-tiigi-docker-4-638.jpg?cb=1505894644